images/win/scripts/Installers/Windows2022/Install-Win10SDK-17763.ps1

```diff
@@ -0,0 +1,9 @@
+################################################################################
+##  File:  Install-Win10SDK-17763.ps1
+##  Desc:  Install Windows 10 SDK (10.0.17763)
+################################################################################
+
+Invoke-WebRequest -Method Get -Uri https://go.microsoft.com/fwlink/p/?LinkID=2033908 -OutFile sdksetup.exe -UseBasicParsing
+Start-Process -Wait sdksetup.exe -ArgumentList "/q", "/norestart", "/ceip off", "/features OptionId.UWPManaged OptionId.UWPCPP OptionId.UWPLocalized OptionId.DesktopCPPx86 OptionId.DesktopCPPx64 OptionId.DesktopCPParm64"
```

Comment:
we have a function that downloads and installs the binary with all the provided flags:
https://github.com/actions/virtual-environments/blob/afc8bd66cc4050c3e0045b18811d572770b7ab4b/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L1